ocs_ci/templates/ceph/ceph-keyring.j2

```diff
@@ -0,0 +1,2 @@
+[client.admin]
+{{ admin_keyring }}
```

Comment:
Add empty line at the end of the file

Reply:
Resolving because the new changes removed this